docs/source/docker/configuration/configuration.rst

```diff
@@ -3,16 +3,14 @@
 Configuration
 ===================================
 
-As the lightly framework the docker solution can be configured using Hydra.
+As the lightly framework the Lightly Worker solution can be configured using Hydra.
```

Comment:
Similar to the Lightly Framework (link), the Lightly Worker solution can be configured using Hydra as well